### PR TITLE
Allows to change the db with an already existing db

### PR DESCRIPTION
**Pull request by LoicCadour**
See description here: https://github.com/medInria/medInria-public/pull/252

### CODE REVIEW

**Comment by mathildemerle** [comment]
Is it needed to be reviewed/tested or you need to still work on it?

**Comment by LoicCadour** [comment]
I need opinion about the idea, feedback, review, every thing.
Thanks

**Comment by mathildemerle** [comment]
I am currently testing this PR. I'm afraid there is a problem under Linux. 

For instance, my DB address is /home/mathildemerle/.local/share/data/INRIA_IHU-LIRYC/MUSIC
And when i click on "Select directory" the browser open /home/mathildemerle/Dev. 

I can't write the address to /home/mathildemerle/.local/share/data/INRIA_IHU-LIRYC and i can't go back in the directory tree. So i needed to find a proposed "/home/mathildemerle" and display hidden files (right click then "Show hidden files"). 

It can be important to add a text saying that changing the database will be ok after restarting if the user use the Save button.

Otherwise, it's working. I changed my database. I had this problem before (when i changed the name of the soft, which is the name of the DB), and i needed to change my current db in a terminal. So, good for me.

In conclusion, can be improved :
1° open the window to  /home/mathildemerle/.local/share/data/INRIA_IHU-LIRYC (means, above the current db). But it can be a problem of hidden files.
2° allow to display hidden files if needed by the address of db.

:)

**Comment by mathildemerle** [comment]
(sorry, i had this problem when i changed "INRIA_IHU-LIRYC", not the same of the soft. So, on my computer, there is two db : inria and INRIA_IHU-LIRYC)

**Comment by LoicCadour** [comment]
I need feedback on the pertinence of this PR to know if I should spend time on improving it or close it.
Thx

**Comment by mikejbuckingham** [comment]
Did you resolve the issue Mathilde had? If so, then I'm happy with this, if it works.

**Comment by LoicCadour** [comment]
Well, I am afraid I can't do much about it (except the mention of Saving after setting the directory).

From my research, showing hidden files cannot be done by qt. On Mac, Cmd+Shift+ .  does the work, on Linux Ctrl + H (according to stackOverflow), ...

Opening the file dialog to the directory above the db is done.

Finally, "go back in directory tree ", I can't find a way to fo that with QFileDialog.

In the meantime, you can write the path in the QLineEdit, so it's a way to overcome these pbs.

**Comment by mathildemerle** [comment]
Ok for me

**Comment by LoicCadour** [comment]
The destination branch is not the correct one. I close it
